bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescription.java

```diff
@@ -113,7 +115,8 @@ public ConfigDescription(URI uri, List<ConfigDescriptionParameter> parameters,
      *
      * @return the URI of this description (not null)
      */
-    public URI getURI() {
+    @Override
+    public URI getUID() {
```

Comment:
I can confirm that this change breaks the openHAB build: https://openhab.ci.cloudbees.com/job/openHAB2-Bundles/99/console